doc/Installation/Installation-(RHEL-CentOS).md

```diff
@@ -122,19 +127,19 @@ Note if not using HTTPd (Apache): RHEL requires `httpd` to be installed regardle
 
 Set `httpd` to start on system boot.
 
-**CentOS 6**
+**CentOS 6: **
     chkconfig --levels 235 httpd on
 
-**CentOS 7**
+**CentOS 7: **
     systemctl enable httpd
```

Comment:
This is broken in the original docs but can you add `` around this please so it appears as a command.